src/base/random.cc

```diff
@@ -46,32 +46,30 @@
 namespace gem5
 {
 
+Random::Random(): Random(globalSeed){}
```

Comment:
Whitespace? I *think* this is a function definition but given that it's all on one line and just above the other definition, I can't tell if that was then intention.